src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/ByteConverter.cs

```diff
@@ -7,12 +7,26 @@ internal sealed class ByteConverter : JsonConverter<byte>
     {
         public override byte Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
         {
+            if (reader.TokenType == JsonTokenType.String &&
+                options != null &&
```

Comment:
I don't think the null check is necessary since we default to Strict (could add an Assert)

Reply:
The logic has been removed due to restructuring to support the attribute.